target/linux/octeon/patches-4.19/102-ubnt-add-erlite-usg3p-dts.patch

```diff
@@ -0,0 +1,59 @@
+--- /dev/null
++++ b/arch/mips/boot/dts/cavium-octeon/ubnt_e100.dts
+@@ -0,0 +1,15 @@
++/*
++ * Device tree source for EdgeRouter Lite.
++ *
++ * Written by: Aaro Koskinen <aaro.koskinen@iki.fi>
++ *
++ * This program is free software; you can redistribute it and/or modify
++ * it under the terms of the GNU General Public License version 2 as
++ * published by the Free Software Foundation.
++ */
++
++/include/ "ubnt_e100-e120.dtsi"
++
++/ {
++	model = "ubnt,e100";
```

Comment:
If we rearrange this anyway, I wonder whether we shouldn't just add a compatible here and use it for the board name in the first place.

Reply:
How do I add a compatible? Is there an example of a compatible device tree I can look at?

Reply:
This means I would patch the ERL device tree to include the Unifi gpio, and have `compatible = ` lines in each block as appropriate?

Reply:
This is about adding a compatible at root node level, describing the device itself and (in most cases) the soc:
https://github.com/openwrt/openwrt/blob/master/target/linux/ath79/dts/ar7241_tplink_tl-wr842n-v1.dts#L10